vault/data_source_kv_secret_test.go

```diff
@@ -27,6 +27,10 @@ func TestDataSourceKVSecret(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, fmt.Sprintf("%s/%s", mount, name)),
 					resource.TestCheckResourceAttr(resourceName, consts.FieldLeaseRenewable, "false"),
+					resource.TestCheckResourceAttr(resourceName, "data.zip", "zap"),
```

Comment:
Probably want to check the length of `data.%`

Reply:
Added in latest commit